### DISCUSSION TITLE
Use wormhole to bridge ETH mainnet to L2 Optimism

### DISCUSSION THREAD

**Discussion by viktorS-dev** [wormhole-foundation/wormhole#1969] [Feature requests] [answered]
Hello team, 

Sorry for the broad question, but the team on Discord could not answer it.
My main question is if it possible to use wormhole to bridge ERC20 to optimism L2 network? https://optimism.io/

Many thanks for your help,

Best,
Viktor

**Comment by leoluk** [comment]
As said on Discord - no, Optimism is not connected to Wormhole. You can bridge tokens to Ethereum and, I suppose, send them to Optimism from there?

**Comment by viktorS-dev** [comment]
Ok thanks, where can I see the full list of chains already integrated with wormhole? thanks

**Comment by leoluk** [comment]
For v1, Ethereum and Solana.

For v2: https://wormholenetwork.com (Ethereum, Terra, BSC, Solana)

**Comment by viktorS-dev** [comment]
Thanks @leoluk ! Right now to transfer funds using the UI of Binance Bridge costs around 25 dollars, will it be cheaper using wormhole?

**Comment by leoluk** [comment]
> Thanks @leoluk ! Right now to transfer funds using the UI of Binance Bridge costs around 25 dollars, will it be cheaper using wormhole?

Maybe, we don't have any official numbers yet. Cost mostly depends on the target chain (i.e. sending messages to Solana is basically free).

**Comment by aadam-10** [comment, marked as answer]
Bridging to optimism L2 is coming soon!
